fusor-ember-cli/app/mirage/config.js

```diff
@@ -56,6 +56,10 @@ export default function() {
     return db.katello_organizations.find(id);
   });
 
+  this.get('/katello/api/v2/organizations/:id/subscriptions', function(db, request) {
+    return {};
```

Comment:
@jmrodri, is this the correct return value {} if there is a subscription for an org?